api/orchestration.go

```diff
@@ -90,8 +90,13 @@ func (o *docDBOrchestrator) documentDBCreate(ctx context.Context, req *DocDBCrea
 		if err = retry(10, 3, 10*time.Second, func() error {
 			msgChan <- fmt.Sprintf("checking if docdb cluster %s is available before continuing", cl)
 
+			if err := o.refreshSession(ctx); err != nil {
+				msgChan <- fmt.Sprintf("unable to refresh orchestrator session: %s", err)
+				return err
```

Comment:
I'm noticing that all of these write to the `msgChan` before returning `err` which writes to `errChan` (which also puts a message on the flywheel log). I don't think it's necessary.

Reply:
ah -- yeah that's right - but do we want to see those in the log?  i guess it doesn't hurt.